nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -94,47 +114,28 @@ def __init__(
             use_cpu_initialization=use_cpu_initialization,
         )
 
-        if activation in ['geglu', 'reglu', 'swiglu']:
-            # Separate linear layer for *GLU activations.
+        if activation == 'geglu':
```

Comment:
Why are we removing the other activations here?

Reply:
Fixed.